data/json/bionics.json

```diff
@@ -706,22 +706,22 @@
     "type": "bionic",
     "name": "Plutonium Purger",
     "included": true,
-    "description": "Emergency reactor fuel purge ejects all fuel from your reactor.",
+    "description": "Triggers an emergency reactor fuel purge that ejects all fuel from your reactor.",
     "act_cost": 1
   },
   {
     "id": "bio_pokedeye",
     "type": "bionic",
     "name": "Bionic Visual Impairment",
-    "description": "Due to a badly misplaced dielectric stylette you are now suffering from mild optic neuropathy.  Increases eye encumbrance by one.",
+    "description": "Due to a badly misplaced dielectric stylette, you are now suffering from mild optic neuropathy.  Increases eye encumbrance by one.",
     "occupied_bodyparts": [ [ "EYES", 1 ] ],
     "flags": [ "BIONIC_FAULTY" ]
   },
   {
     "id": "bio_power_armor_interface",
     "type": "bionic",
     "name": "Power Armor Interface",
-    "description": "Interfaces your power system with the internal charging port on suits of power armor.",
+    "description": "Interfaces your power system with the internal charging port on suits of power armor, allowing them to be activated.",
```

Comment:
You can activate power amor without this by using UPS, so something like "allowing them to draw from your bionic power" might be better.